api/core/v1beta1/conversion.go

```diff
@@ -884,3 +884,31 @@ func Convert_v1beta1_MachinePoolStatus_To_v1beta2_MachinePoolStatus(in *MachineP
 func Convert_v1beta1_MachinePoolSpec_To_v1beta2_MachinePoolSpec(in *MachinePoolSpec, out *clusterv1.MachinePoolSpec, s apimachineryconversion.Scope) error {
 	return autoConvert_v1beta1_MachinePoolSpec_To_v1beta2_MachinePoolSpec(in, out, s)
 }
+
```

Comment:
Am I missing something or should we have the manual conversions here (copying over from old metadata <-> deprecatedMetadata )?